quicksort/Cpp/quick_sort.cpp

```diff
@@ -0,0 +1,42 @@
+
+#include <iostream>
+
+using std::cout;
+
+void quick_sort(int a[], int left, int right) {
+
+      int i = left, j = right;
+      int pivot = a[right];
+
+      while (i <= j) {
+        while (a[i] < pivot)
+            i++; //carry on, this part already sorted
+        while (a[j] > pivot)
+            j--; //carry on, this part already sorted
+        if (i <= j) { // a[i]>=pivot and a[j]<=pivot
+            //swap a[i] with a[j]
+            int tmp = a[i];
+            a[i] = a[j];
+            a[j] = tmp;
+            i++;
+            j--;
+        }
+      };
```

Comment:
You don't need a ; after a }  [readability/braces] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*